README.md

```diff
@@ -200,6 +200,58 @@ pipenv shell
 apt install lintian
 make test
 ```
+## Signing sources
+
+SecureDrop Workstation code spans across the following repositories:
+
+https://github.com/freedomofpress/securedrop-client
+https://github.com/freedomofpress/securedrop-debian-packaging
+https://github.com/freedomofpress/securedrop-proxy
+https://github.com/freedomofpress/securedrop-sdk
+https://github.com/freedomofpress/securedrop-workstation
+https://github.com/freedomofpress/qubes-template-securedrop-workstation
```

Comment:
These URLs must be formatted as a bulleted list, otherwise Markdown will concatenate them into a paragraph.